addons/tmf_loadouts/loadouts/UK_Army_2005.hpp

```diff
@@ -0,0 +1,544 @@
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	vest[] = {"CUP_V_B_BAF_DDPM_Osprey_Mk3_AutomaticRifleman"};
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	primaryWeapon[] = {"CUP_lmg_minimipara"};
+	sidearmWeapon[] = {"hlc_pistol_P226US"};
+	magazines[] = {
+		LIST_4("CUP_200Rnd_TE4_Red_Tracer_556x45_L110A1"),
+		"CUP_HandGrenade_L109A1_HE",
+		"SmokeShell",
+		LIST_3("hlc_15Rnd_9x19_B_P226")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backPack[] = {"usm_pack_200rnd_556_bandoliers"};
+	backpackItems[] = {
+		LIST_4("CUP_200Rnd_TE4_Red_Tracer_556x45_L110A1"),
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : car {
+	displayName = "Rifleman (AT)";
+	secondaryWeapon[] = {"CUP_launch_NLAW"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	primaryWeapon[] = {"CUP_srifle_AWM_wdl"};
+	scope[] = {"hlc_optic_zf95base"};
+	bipod[] = {"cup_bipod_harris_1a2_l_blk"};
+	secondaryWeapon[] = {"hlc_pistol_P226US"};
+	magazines[] = {
+		LIST_9("CUP_5Rnd_86x70_L115A1"),
+		LIST_5("hlc_15Rnd_9x19_B_P226"),
+		LIST_2("CUP_HandGrenade_L109A1_HE"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	primaryWeapon[] = {"CUP_lmg_FNMAG"};
+	backPack[] = {"usm_pack_762x51_bandoliers"};
+	magazines[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_Red_Tracer_762x51_Belt_M"),
+		"CUP_HandGrenade_L109A1_HE",
+		"SmokeShell",
+		LIST_3("hlc_15Rnd_9x19_B_P226")
+	};
+};
+
+class mmgac : r {
+	displayName = "MMG Ammo Carrier";
+	backPack[] = {"CFP_Carryall_DPMWoodland"};
+	backpackItems[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_Red_Tracer_762x51_Belt_M")
+	};
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	backPack[] = {"usm_pack_762x51_bandoliers"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_3("CUP_100Rnd_TE4_LRT4_Red_Tracer_762x51_Belt_M")
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CFP_Carryall_DPMWoodland"};
+	secondaryWeapon[] = {"ace_csw_staticM2ShieldCarry"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+};
+
+class hmgac : r {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CFP_Carryall_DPMWoodland"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripod"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CFP_Carryall_DPMWoodland"};
+	secondaryWeapon[] = {"ace_csw_m3CarryTripodLow"};
+	backpackItems[] = {
+		LIST_3("ace_csw_100Rnd_127x99_mag_red")
+	};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	secondaryWeapon[] = {"CUP_launch_NLAW"};
```

Comment:
Either Milan or Carl Gustav

Reply:
Lets go with Milan, that should be fun/interesting right?